tx/textual/valuerenderer/enum.go

```diff
@@ -0,0 +1,27 @@
+package valuerenderer
+
+import (
+	"context"
+	"io"
+
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+type enumRenderer struct{}
```

Comment:
Let's also  add an exported `NewDecValueRenderer` function, like in other renderers

Reply:
Roger that!